main.go

```diff
@@ -94,6 +94,7 @@ func main() {
 		Development: true,
 	}
 	opts.BindFlags(flag.CommandLine)
+	klog.InitFlags(nil)
```

Comment:
I'm, not sure the argument is correct

Reply:
Same as in PAO https://github.com/openshift-kni/performance-addon-operators/blob/master/main.go#L89

Reply:
OK, we do what I'd expect inside the function itself: https://github.com/Tal-or/numaresources-operator/blob/387e57987d0485850698e61239c75d1e3adc7f9e/vendor/k8s.io/klog/v2/klog.go#L425